### PR TITLE
New vignette + doc edits

### PR DESCRIPTION
**Pull request by jgabry**
This PR adds a package overview vignette plus a bunch of minor doc edits. Of the 72 files changed only 2 have substantive changes (the others are minor grammar/punctuation/phrasing edits in Rd files with no real changes of consequence). The 2 files to look at are: 

* `convergence.R`
* `vignettes/posterior.Rmd`

In `convergence.R` I added an overview doc page with a list of the available convergence diagnostics and links to their individual help pages. The downside to this is that we need to remember to keep it up to date, but I do think it's nice to have. 

The vignette `posterior.Rmd` takes the examples from the readme and fleshes them out and adds a few more sections. In the (potentially near) future I still think it probably makes sense to split the diagnostics out into their own vignette like we mentioned in #42 but I haven't done that yet. Feel free to point out things you don't like about it and I'm happy to make changes (or feel free to just edit it if you prefer). 

I also added links between the new vignette and the existing rvar vignette in both directions. 

We might consider closing #42 after this and opening a new issue for vignette improvements. I think that we could probably get new contributors to help with that if we mark it as "help wanted" or "good first issue" since making improvements to existing vignettes is a less daunting task than creating the vignette in the first place.

### CODE REVIEW

**Comment by jgabry** [comment]
Also, once I incorporate any review feedback we should probably regenerate the website, which will include the two vignettes (I updated pkdown.yaml to enable that) and include the new doc for the parallelized `summarise_draws()`. I suppose we should bump the the version number to 0.1.6 when we do that.

File: vignettes/rvar.Rmd [hunk 1]
```diff
@@ -96,9 +96,10 @@ post
 ```
 
 The `draws_rvars()` datatype works much the same way that other `draws` formats
-do; see **TODO: INSERT LINK** for an introduction to `draws` objects. One difference is that
-`draws_rvars` counts variables differently, because it allows variables to be multidimensional.
-For example, the `post` object above contains two variables, `mu` and `Sigma`:
+do; see the main [posterior vignette](posterior.html) for an introduction to
```
Comment (mjskay) [inline vignettes/rvar.Rmd:99]: I don't know all the pros/cons; is this vignette inter-linking format preferable to `vignette("posterior")`, which will be auto-linked by {pkgdown}?

Comment (jgabry) [reply vignettes/rvar.Rmd:99]: I didn't realize pkgdown would auto link it. We should just use `vignette("posterior")` in that case.

Comment (jgabry) [reply vignettes/rvar.Rmd:99]: Will change it in both vignettes

**Review by mjskay — APPROVED** [review]
Awesome, this looks great to me!

**Comment by paul-buerkner** [comment]
Looks good to me too! @jgabry feel free to merge!

**Comment by jgabry** [comment]
Ok great, I'll merge and then regenerate the website

**Comment by jgabry** [comment]
We're live: http://mc-stan.org/posterior/articles/index.html

**Comment by jgabry** [comment]
I also added a section to the function reference that includes all the rvar-specific stuff (this was missing before): 

http://mc-stan.org/posterior/reference/index.html#section-functionality-specific-to-the-rvar-datatype

**Comment by paul-buerkner** [comment]
thank you!

Jonah Gabry ***@***.***> schrieb am Sa., 22. Mai 2021, 18:57:

**Comment by mjskay** [comment]
Looks great!

**Pull request merged**